packages/react-composites/src/composites/CallComposite/pages/NoticePage.tsx

```diff
@@ -52,7 +57,18 @@ export function NoticePage(props: NoticePageProps): JSX.Element {
         </Text>
         {!props.disableStartCallButton && (
           <Stack styles={rejoinCallButtonContainerStyles}>
-            <StartCallButton onClick={() => adapter.joinCall()} disabled={false} rejoinCall={true} autoFocus />
+            <StartCallButton
+              onClick={() => {
+                if (callees) {
```

Comment:
callees is `as StartCallIdentifier[]`, this means we should be checking length here as technically callees will always be valid

Reply:
Otherwise it should be `as StartCallIdentifier[] | undefined`